integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -2700,23 +2728,40 @@ msgstr "Nutzer-Chats"
 msgid "name"
 msgstr "Name"
 
-#: cms/models/contact/contact.py
-msgid "POI"
+#: cms/models/contact/contact.py cms/models/events/event.py
+#: cms/models/pois/poi.py cms/models/pois/poi_translation.py
+msgid "location"
 msgstr "Ort"
 
 #: cms/models/contact/contact.py cms/models/pois/poi.py
 msgid "email address"
 msgstr "E-Mail-Adresse"
 
+#: cms/models/contact/contact.py
+msgid "E-mail from location"
+msgstr "E-mail vom Ort"
+
+#: cms/models/contact/contact.py
+msgid "Will be taken over from the location."
+msgstr "Wird vom Ort übernommen."
+
 #: cms/models/contact/contact.py cms/models/pois/poi.py
 msgid "phone number"
 msgstr "Telefonnummer"
 
+#: cms/models/contact/contact.py
+msgid "phone number from location"
+msgstr "Telefonnummer vom Ort"
+
 #: cms/models/contact/contact.py cms/models/pois/poi.py
 #: cms/models/users/organization.py
 msgid "website"
 msgstr "Webseite"
 
+#: cms/models/contact/contact.py
+msgid "website from location"
+msgstr "Website vom Ort"
```

Comment:
I thing this might be a bit more clear, but I'm not sure. Just *E-mail vom Ort* seems a bit weird to me
```suggestion
#: cms/models/contact/contact.py
msgid "E-mail from location"
msgstr "E-mail des Ortes"

#: cms/models/contact/contact.py
msgid "Will be taken over from the location."
msgstr "Wird vom Ort übernommen."

#: cms/models/contact/contact.py cms/models/pois/poi.py
msgid "phone number"
msgstr "Telefonnummer"

#: cms/models/contact/contact.py
msgid "phone number from location"
msgstr "Telefonnummer des Ortes"

#: cms/models/contact/contact.py cms/models/pois/poi.py
#: cms/models/users/organization.py
msgid "website"
msgstr "Webseite"

#: cms/models/contact/contact.py
msgid "website from location"
msgstr "Website des Ortes"
```